### PR TITLE
i18n(es): Translate `caisy.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translate `caisy.mdx`

### CODE REVIEW

File: src/content/docs/es/guides/cms/caisy.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+---
+<h1>{post.title}</h1>
+<RichTextRenderer node={post.text.json} />
+```
+
+## Recursos oficiales
+
+- Mira el ejemplo de Caisy + Astro en [GitHub](https://github.com/caisy-io/caisy-example-astro) o [Stackblitz](https://stackblitz.com/github/caisy-io/caisy-example-astro?file=src%2Fpages%2Fblog%2F%5B...slug%5D.astro)
+- Realiza consultas a tus documentos en [modo borrador](https://caisy.io/developer/docs/external-api/localization-and-preview#preview-mode-15) y múltiples [idiomas](https://caisy.io/developer/docs/external-api/localization-and-preview#localization-in-a-graphql-query-8).
+- Utiliza [paginación](https://caisy.io/developer/docs/external-api/queries-pagination) para consultar grandes cantidades de documentos.
```
Comment (dreyfus92) [inline src/content/docs/es/guides/cms/caisy.mdx:61]:
```suggestion
- Utiliza [paginación](https://caisy.io/developer/docs/external-api/queries-pagination) para consultar un gran número de documentos.
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo @Waxer59, te dejo una sugerencia 😁

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**